packages/edit-site/src/components/save-hub/index.js

```diff
@@ -32,29 +55,106 @@ export default function SaveHub() {
 		[]
 	);
 
+	const {
+		editEntityRecord,
+		saveEditedEntityRecord,
+		__experimentalSaveSpecifiedEntityEdits: saveSpecifiedEntityEdits,
+	} = useDispatch( coreStore );
+
 	const disabled = isSaving || ( ! isDirty && ! isPreviewingTheme() );
 
+	// if we have only one unsaved change and it matches current context, we can show a more specific label
+	let dirtyLocal = null;
```

Comment:
Wondering if `dirtyCurrentEntity` or something like that would be a more self-explanatory variable name since then it'd align with `saveCurrentyEntity`.